### DISCUSSION TITLE
As the amount of solar energy goes up, so does my home usage.

### DISCUSSION THREAD

**Discussion by Stephen2615** [jasonacox/Powerwall-Dashboard#522] [General]
Greetings,

I have noticed this quite often and don't understand what is going on. The Power Flow graphic says something like 7.5 kWh generated with only 1.5 kWh going to the grid. I know my house is not using anything like 6 kWh. With the normal equipment running, I think I use about 2 kWh.  The battery is fully charged. Does anyone else see this behaviour? 

Did the installers do something wrong? I will download the raw usage from my provider and see if my grid export is the same as what is shown in the dashboard. 

TIA!

Stephen

**Comment by Stephen2615** [comment]
An update. 
My electricity provider says I exported 36.1 kWh yesterday. 
The Dashboard says I sent 6.8 kWh to the grid. 
My provider says I used 13.98 kWh from the grid, thanks to my three phase Air Conditioner.  
My Inverter says I created 45.9 kWh where the Dashboard says 46.5 (close enough). 
My figures say I consumed 9.8 kWh of our own power, probably through the smart meter (solar or battery).  
Our provider's figures have total power we consumed (our solar power and the grid) says 23.78 kWh. 
The Dashboard says Home Usage was 39.1 kWh. 
I have no idea on how to determine how much of the battery power we use but my Home Assistant numbers have battery export as 3.6 kWh and battery import as 4.2 kWh. 
The Dashboard has To Powerwall as 5.0 kWh and From Powerwall as 3.9 kWh. 
Interestingly, while watching the dashboard, clouds came over the solar power dropped to 2 kWh and my home usage dropped to 1.7 kWh and we are exporting to the grid.
With full sun, a few minutes ago, home usage was 6 kWh. 
The more solar power, the more the house uses it and we are still exporting to the grid. 

Something is not right..

**Comment by mcbirse** [comment]
> Did the installers do something wrong?

Hi @Stephen2615 - I'd say, yes, highly likely the installers have misconfigured your system. It sounds like your CT's are not connected correctly.

To eliminate potential Powerwall-Dashboard issue, do you see the same problems in the official Tesla app?

If yes, go back to your installers and get them to check how they have wired your CT's.

**Comment by Stephen2615** [comment]
Hi @mcbirse 

Thanks. The Tesla downloadable data matches the Dashboard so me thinks the installer has mucked up something. I had the backup feature installed last year and he had all sorts of issues getting it right. Time to give them a call. 

Cheers

Stephen

**Comment by Stephen2615** [comment]
Hi @mcbirse.

Tesla said the installer has used US wiring instead of EU (same as Australia) wiring. It will soon be fixed. 

Stephen

**Reply by jasonacox** [reply]
That sounds important! 😉 It's good you found this and I hope the Dashboard help.
